README.md

```diff
@@ -8,6 +8,17 @@ A Python client for the [Snyk API](https://snyk.docs.apiary.io/#).
 
 Using the client requires you to provide your Snyk API token.
 
+```bash
+pip3 install snyk
```

Comment:
This is a poetry project. I would also recommend including the appropriate poetry command as well 
`poetry install pysnyk`